pkg/cmd/ssh/options.go

```diff
@@ -419,18 +421,29 @@ func (o *SSHOptions) Run(f util.Factory) error {
 		return err
 	}
 
-	if currentTarget.ShootName() == "" {
-		return errors.New("no Shoot cluster targeted")
-	}
-
-	printTargetInformation(o.IOStreams.Out, currentTarget)
-
 	// create client for the garden cluster
 	gardenClient, err := manager.GardenClient(currentTarget.GardenName())
 	if err != nil {
 		return err
 	}
 
+	if currentTarget.ShootName() == "" {
```

Comment:
same here. The currentTarget can only be chnaged via the target command.

Reply:
I renamed the current target